platforms/bukkit/common/build.gradle.kts

```diff
@@ -1,12 +1,17 @@
 repositories {
-
+    maven {
+        name = "onarandombox"
+        url = uri("https://repo.onarandombox.com/content/groups/public/")
+    }
```

Comment:
Rather than adding a new maven repository, I think it would be best if I proxied it through https://maven.solo-studios.ca

what is `/content/groups/public`?
because, no such repository and/or group id seems to exist in the reposilite instance.

I'm assuming I'd want to proxy the `multiverse-releases` repo with the group id `org.mvplugins`?

Reply:
at the moment it seems we're not using my repo, but eventually™ when we drop the majority of repos for that, I'll make sure to proxy it properly

in the meantime though we should probably use `https://repo.onarandombox.com/multiverse-releases`